wallet/concepts/apis.md

```diff
@@ -22,6 +22,13 @@ This API is specified by [EIP-1193](https://eips.ethereum.org/EIPS/eip-1193), an
 request users' Ethereum accounts, read data from blockchains the user is connected to, suggest
 that the user sign messages and transactions, and more.
 
+:::note
+MetaMask supports [EIP-6963](https://eips.ethereum.org/EIPS/eip-6963), which introduces an
+alternative discovery mechanism to the `window.ethereum` injected provider, enabling dapps to
```

Comment:
i think we should mention somewhere that this is the new and preferred pattern for provider discovery

Reply:
and/or emphasize that `window.ethereum` injection is deprecated (but in reality, we won't be removing this for years if ever)